packages/Babylonian-Printbugger.package/BPPrintbugger.class/instance/newColumnFor..st

```diff
@@ -0,0 +1,17 @@
+private -ui 
+newColumnFor: aBPExample
+
+	| columnMorph |
+	columnMorph := Morph new
```

Comment:
Wäre es nicht besser columnMorph in eine eigene Klasse auszulagern, die sich zB. das displayed Example auch gleich speichern kann?

Reply:
Finde eine eigene Klasse atm noch zu overengineered, da es gerade nur ein wrapper für die swimlane ist. mit mehr buttons in einer reihe könnte ich es mir vorstellen.